.clang-format

```diff
@@ -1,5 +1,5 @@
 ---
-Language:        Cpp
+# Language: C
```

Comment:
A new version of clang-format (>= 20) did complain on either C or Cpp not being supported. Remedy was to comment it out...